tests/test_algorithm.py

```diff
@@ -1669,69 +1395,52 @@ def make_equity_info(cls):
 
     @classmethod
     def make_equity_daily_bar_data(cls):
-        days = len(cls.equity_daily_bar_days)
-        return trades_by_sid_to_dfs(
-            {
-                0: factory.create_trade_history(
-                    0,
-                    [10.0] * days,
-                    [100] * days,
-                    timedelta(days=1),
-                    cls.sim_params,
-                    cls.trading_calendar),
-                3: factory.create_trade_history(
-                    3,
-                    [10.0] * days,
-                    [100] * days,
-                    timedelta(days=1),
-                    cls.sim_params,
-                    cls.trading_calendar)
-            },
-            index=cls.equity_daily_bar_days,
-        )
+        cal = cls.trading_calendars[Equity]
+        sessions = cal.sessions_in_range(cls.START_DATE, cls.END_DATE)
+        frame = pd.DataFrame({
```

Comment:
The original impetus for rewriting this method was removing the dependency on `cls.sim_params` (see note above), but I think this also makes it much clearer what the price data is in this suite.